application/api/testsuite.py

```diff
@@ -93,6 +96,223 @@ def get(self, session):
+                                        else:
+                                            column[columnpair] = columnpair
+                                    testcasedetail["column"] = column
+                                elif ";" in user_test_case_detail["column"]:
+                                    column = {}
+                                    columns = user_test_case_detail[
+                                        "column"].split(";")
+                                    for singlecolumn in columns:
+                                        column[singlecolumn] = singlecolumn
+                                    testcasedetail["column"] = column
+                                else:
+                                    column = {}
+                                    column[user_test_case_detail["column"]] = \
+                                        user_test_case_detail["column"]
+                                    testcasedetail["column"] = column
+                                db_obj.save_to_db()
+                    db_obj.test_case_detail = testcasedetail
+                    db_obj.save_to_db()
+                    return api_response(
+                        True, APIMessages.TEST_SUITE_UPDATED.format(
+                            test_case_id), STATUS_CREATED)
+                else:
+                    return api_response(False,
+                                        APIMessages.TEST_CASE_NOT_IN_DB.format(
+                                            test_case_id),
+                                        STATUS_BAD_REQUEST)
+            else:
+                return api_response(False, APIMessages.PASS_TESTCASEID,
+                                    STATUS_BAD_REQUEST)
+
+        except SQLAlchemyError as e:
+            db.session.rollback()
+            return api_response(False, APIMessages.INTERNAL_ERROR,
+                                STATUS_SERVER_ERROR,
+                                {'error_log': str(e)})
+        except Exception as e:
+            return api_response(False, APIMessages.INTERNAL_ERROR,
+                                STATUS_SERVER_ERROR,
+                                {'error_log': str(e)})
+
+
+class CreateNewTestSuite(Resource):
+    """ To handle POST API to create a new test suite."""
+
+    @token_required
+    def post(self, session):
+        """
+            To create a new test suite from the given test case list.
+
+             Args:
+                 session (object):By using this object we can get the user_id.
+
+             Returns:
+                 Standard API Response with message(returns message saying
+                 that New Test Suite created),http status code.
+        """
+        parser = reqparse.RequestParser()
+        parser.add_argument('case_id_list',
+                            help=APIMessages.PARSER_MESSAGE,
+                            required=True, type=args_as_list, default=[])
```

Comment:
Instead of args as list, can we use list?

Reply:
ya i can change it.